### PR TITLE
DOC Add additional info to 4.9.0 changelog

### PR DESCRIPTION
**Pull request by maxime-rainville**
This PR folds back some of the ad doc changelog tweaks we added to the 4.9.0-beta1 release. It also adds a bit to say that the release has been audited.

### CODE REVIEW

**Comment by maxime-rainville** [comment]
Added a bit about the EOL of IE 11

File: docs/en/04_Changelogs/4.9.0.md [hunk 1]
```diff
@@ -2,13 +2,38 @@
 
 ## Overview
 
+- [Security audit and regression test](#audit)
+- [For development teams of Common Web Platform projects](#cwp-end)
+- [Dropping support for Internet Explorer 11](#ie11eol)
```
Comment (maxime-rainville) [inline docs/en/04_Changelogs/4.9.0.md:7]: Maybe this stuff should be after "Features and enhancements"

**Review by brynwhyman — CHANGES_REQUESTED** [review]
It doesn't look like this is following the usual changelog template. Do you mind updating this to be in line? See: https://github.com/silverstripe/recipe-kitchen-sink/blob/4/.cow/changelog.md.twig

**Comment by maxime-rainville** [comment]
The file you pointed to is based off the template for the CWP changelog. The CWP change log gets generated from scratch for each release. The installer changelog however is meant to be updated as new headline feature are added.

Not saying one approach is better than the other. But the current format is the status quo from the core releases point of view.

I've updated the PR to include bits that seem relevant. Other stuff like the known failures or know issues, I'm not quite sure what to do with.

File: docs/en/04_Changelogs/4.9.0.md [hunk 2]
```diff
@@ -2,13 +2,130 @@
...
+<details>
+<summary>Core module versions</summary>
+
+| Module | Version |
+| ------ | ------- |
+| silverstripe/admin | 1.9.0 |
+| silverstripe/asset-admin | 1.9.0 |
+| silverstripe/assets | 1.9.0 |
+| silverstripe/campaign-admin | 1.9.0 |
+| silverstripe/cms | 1.9.0 |
```
Comment (brynwhyman) [inline docs/en/04_Changelogs/4.9.0.md:17]:
```suggestion
| silverstripe/cms | 4.9.0 |
```

File: docs/en/04_Changelogs/4.9.0.md [hunk 3]
```diff
@@ -2,13 +2,130 @@
...
     - [Other new features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Security audit and regression test{#audit}
+
+This release has gone through a regression test suite to identify any possible instability between modules.
+
+The release has also been audited by an independant security firm to identify possible security regressions.
+
+Review the [security audit report](INSERT-LINK-TO-AUDIT).
```
Comment (brynwhyman) [inline docs/en/04_Changelogs/4.9.0.md:114]:
```suggestion
## Regression test and Security audit{#audit}

This release has been comprehensively regression tested and passed to a third party for a security-focused audit. 

While it is still advised that you perform your own due diligence when upgrading your project, this work is performed to ensure a safe and secure upgrade with each recipe release.
```

Comment (brynwhyman) [reply docs/en/04_Changelogs/4.9.0.md:114]: I agree that we should look to provide the audit report however we're not in a position to do this now. Based on my experience, the report and vendor response can require follow-up conversations which will need some further thinking on how to support that in a channel like this.

In lieu of a report, I've tried to tweak the wording to explain the benefit of this in another way.

**Review by brynwhyman — CHANGES_REQUESTED** [review]

File: docs/en/04_Changelogs/4.9.0.md (hunk 1 above)
Comment (brynwhyman) [reply docs/en/04_Changelogs/4.9.0.md:99]: I think it's fine where it is

File: docs/en/04_Changelogs/4.9.0.md (hunk 2 above)
Comment (maxime-rainville) [reply docs/en/04_Changelogs/4.9.0.md:17]: Copied that from the RC change log, so I probably borked it there as well.

**Review by brynwhyman — APPROVED** [review]

**Comment by brynwhyman** [comment]
> The file you pointed to is based off the template for the CWP changelog. The CWP change log gets generated from scratch for each release. The installer changelog however is meant to be updated as new headline feature are added.

I thought I commented this already, but must have not published it...

There is a ss-installer changelog template as well, [see this](https://github.com/silverstripe/silverstripe-installer/blob/4/.cow/changelog.md.twig). I think the issue is that the template won't be used if there's already a changelog file present, which is often the case if you're waiting until the beta to use cow to generate the changelog.

My preference is definitely to use a template to generate the changelog early on. That wording has previously been approved and it keeps the messaging consistent - and avoids the rework that is often necessary when it's not used. For example, the Known Issues was agreed to be used when a release was known to contain a critical regression that couldn't have been avoided, or was found after the fact. 

Now that there's a single release, I'm guessing it's worth a review on how this changelog is created and see if it can be worked into a process that works. Something for another issue anyway!

**Comment by maxime-rainville** [comment]
Having the template there is kind of pointless because cow only used it if there's not a pre-existing changelog file. But if we follow the normal process, we'll always have a pre-existing changelog file for the next minor by the time we run cow.

We probably just need to have a post stable task that is "Create a changelog entry for next minor from template".

**Pull request merged**